custom_components/auto_backup/config_flow.py

```diff
@@ -45,7 +45,7 @@ def async_get_options_flow(config_entry):
 class OptionsFlowHandler(config_entries.OptionsFlow):
     def __init__(self, config_entry):
         """Initialize Auto Backup options flow."""
-        self.config_entry: config_entries.ConfigEntry = config_entry
+        self.config_entry = config_entry
```

Comment:
This line can be removed now from what I can tell, I think the assignment will still trigger the warning.

https://developers.home-assistant.io/blog/2024/11/12/options-flow/